### ISSUE TITLE
Support:  Where did the .git/info/exclude window go?

### ISSUE THREAD

**Issue by simonschaufi** [hsz/idea-gitignore#476]
### Prerequisites

* [X] Plugin is in the latest version
* [X] Issue was not reported yet
* [ ] Stack trace (if provided) contains `mobi.hsz.idea.gitignore` package name

**Expected behavior:**

PhpStorm shows `.git/info/exclude` window when editing .gitignore file

**Actual behavior:**

Since update to latest version the window is not showing up any more and I didn't find a setting for that.

**Reproduces how often:**

always

### Versions

**Plugin:**  

```
2.2.1
```

**IDE:** 

```
PhpStorm 2017.2.2
Build #PS-172.3968.35, built on August 31, 2017
JRE: 1.8.0_152-release-915-b11 amd64
JVM: OpenJDK 64-Bit Server VM by JetBrains s.r.o
Windows 10 10.0
```

**OS:** 

```
Windows 10
```

**Comment by hsz** [comment]
@simonschaufi Thanks for this report. Indeed there was a problem with caching those files - it's fixed and will be released with `v2.3.0`

**Comment by simonschaufi** [comment]
Thank you for fixing it! 😍
